src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -172,14 +203,29 @@ class EduSharingConnector {
 			return new EduSharingResponse();
 		}
 
-		const criterias = [];
-		if (Configuration.get('FEATURE_ES_MERLIN_ENABLED')) {
-			const county = await getCounty(schoolId);
-			const groups = ['GROUP_public', 'GROUP_LowerSaxony-public', 'GROUP_Thuringia-public'];
-			if (county && county.countyId) {
-				groups.push(`GROUP_county-${county.countyId}`);
-			}
+		/* Providing the appropriate permissions, i.e., which items we are allowed to access, by specifying 
+			the necessary groups. */
 
+		const countyGroups = []; // County permissions.
+		const county = await getCounty(schoolId);
+		if (county && county.countyId) {
+			countyGroups.push(`GROUP_county-${county.countyId}`);
+		}
+
+		let instancePermissions = edusharingInstancePermissions.find(
```

Comment:
maybe a better name would be currentInstance

Reply:
currentInstancePermissions ?